frame/contracts/src/lib.rs

```diff
@@ -266,7 +266,7 @@ pub type BalanceOf<T> =
 pub type NegativeImbalanceOf<T> =
 	<<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::NegativeImbalance;
 
-pub trait Config: frame_system::Config {
+	pub trait Config: frame_system::Config {
```

Comment:
Is it right formatting?

Reply:
No, but it's a Substrate team mistake. I'll leave it to avoid future conflicts.